test/test_nn.py

```diff
@@ -3110,7 +3111,8 @@ def run_tests(dtype, mode, sparse, trainable_per_sample_weights):
     def test_EmbeddingBag_per_sample_weights_and_no_offsets(self):
         self._test_EmbeddingBag_per_sample_weights_and_no_offsets(self)
 
-    @unittest.skipIf(not TEST_CUDA, "CUDA unavailable")
+    @unittest.skip("temporarily disable until failures are fixed. t54369166")
```

Comment:
@smessmer @jerryzh168 In the future, please open a github issue wherever possible instead of a fb-internal task for external visibility. A user or external contributor who is looking through the tests won't be able to see the context that is hidden in an internal-only task.